js/graph_config_dialog.js

```diff
@@ -323,7 +324,7 @@ function GraphConfigurationDialog(dialog, onSave) {
                     },
                     color: $('select.color-picker option:selected', this).val(),
                     lineWidth: parseInt($("input[name=linewidth]", this).val()),
-                    grid: (($('div#grid', this).attr("value")==="true")?true:false)
+                    grid: (($('input[name=grid]', this).is(':checked')))
```

Comment:
To make Sonar happy:
- remove the useless extra parenthesis
- add a trailing comma to the last element of the list
This issues were there before your change, but now that you are modifying the line... 🤪

Reply:
Pushed change